drivers/adc/adc_handlers.c

```diff
@@ -7,8 +7,21 @@
 #include <adc.h>
 #include <syscall_handler.h>
 
-_SYSCALL_HANDLER1_SIMPLE_VOID(adc_enable, K_OBJ_DRIVER_ADC, struct device *);
-_SYSCALL_HANDLER1_SIMPLE_VOID(adc_disable, K_OBJ_DRIVER_ADC, struct device *);
+_SYSCALL_HANDLER(adc_enable, dev)
```

Comment:
well, those APIs are going away AFAIK, there is a new ADC API being proposed....

Reply:
strongly concur with @lpereira